pyproject.toml

```diff
@@ -28,25 +28,26 @@ dependencies = [
     "lxml",
     "matplotlib~=3.5.1",
     "nltk~=3.7",
-    "numpy~=1.19.2",
+    "numpy",
     "omegaconf~=2.2.2",
     "PyPDF2~=2.10.7",
     "python_magic~=0.4.18",
-    "pytorch_lightning~=1.7.1",
-    "requests~=2.21.0",
+    "pytorch_lightning~=2.0.4",
```

Comment:
What are the rules that guiding these version changes?

Reply:
Try the best not to change the dependecies. But due to training purpose(e.g. LLM training strategies), packages related to training maybe newer and be specified more accurately.